### ISSUE TITLE
Add FreeBSD support in Fresh

### ISSUE THREAD

**Issue by GuillaumeBibaut** [denoland/fresh#309]
Hello
I've already contacted `evanw/esbuild` about this subject. FreeBSD is supported inside esbuild, and I've asked for FreeBSD support in the `esbuild/deno_esbuild` 3rd party package. Unless impossible, could you please look into some FreeBSD support inside Fresh please?

The actual deno `my-project` repo for tutorial purpose throws an exception on FreeBSD:

```
$ deno task start
Warning deno task is unstable and may drastically change in the future
Task start deno run -A --watch=static/,routes/ dev.ts
Watcher Process started.
The manifest has been generated for 3 routes and 1 islands.
Server listening on http://localhost:8000
An error occured during route handling or page rendering. Error: Unsupported platform: x86_64-unknown-freebsd
    at install (https://deno.land/x/esbuild@v0.14.39/mod.js:1791:11)
    at https://deno.land/x/esbuild@v0.14.39/mod.js:1800:29
    at ensureServiceIsRunning (https://deno.land/x/esbuild@v0.14.39/mod.js:1928:7)
    at Module.initialize (https://deno.land/x/esbuild@v0.14.39/mod.js:1696:9)
    at ensureEsbuildInialized (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:14:15)
    at Bundler.bundle (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:43:11)
    at Bundler.cache (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:88:26)
    at Bundler.get (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:97:30)
    at Object.any (https://deno.land/x/fresh@1.0.0/src/server/context.ts:545:40)
    at https://crux.land/api/get/uYQG.ts:163:40
error: Error: Unsupported platform: x86_64-unknown-freebsd
    at install (https://deno.land/x/esbuild@v0.14.39/mod.js:1791:11)
    at https://deno.land/x/esbuild@v0.14.39/mod.js:1800:29
    at ensureServiceIsRunning (https://deno.land/x/esbuild@v0.14.39/mod.js:1928:7)
    at Module.initialize (https://deno.land/x/esbuild@v0.14.39/mod.js:1696:9)
    at ensureEsbuildInialized (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:14:15)
    at Bundler.bundle (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:43:11)
    at Bundler.cache (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:88:26)
    at Bundler.get (https://deno.land/x/fresh@1.0.0/src/server/bundle.ts:97:30)
    at Object.any (https://deno.land/x/fresh@1.0.0/src/server/context.ts:545:40)
    at https://crux.land/api/get/uYQG.ts:163:40
```

**Comment by lucacasonato** [comment]
Deno does not officially support FreeBSD. If there is something that we can do in esbuild to make this work, that's great. I am happy to merge low invasiveness PRs in Fresh if it unblocks FreeBSD, but I won't be working on this myself as FreeBSD support is not a priority for me.
